src/services/ServiceLocator.ts

```diff
@@ -42,9 +43,13 @@ class ServiceLocator {
   private eventHandler?: SparrowEventHandler;
 
   constructor() {
-    this.prisma = Config.databaseURL
-      ? getPrismaClient(Config.databaseURL)
+    const notehubProvider = Config.notehubProvider;
+    const databaseURL = Config.databaseURL;
+    this.prisma = !notehubProvider
+      ? getPrismaClient(databaseURL)
       : undefined;
+    const msg = this.prisma ? `Connecting to database at ${databaseURL}` : 'Using Notehub provider';
+    serverLogInfo(msg);
```

Comment:
```suggestion
    const message = this.prisma ? `Connecting to database at ${databaseURL}` : 'Using Notehub provider';
    serverLogInfo(message);
```
force a rebuild on netlify perhaps